src/Cms/Files.php

```diff
@@ -3,10 +3,22 @@
 namespace Kirby\Cms;
 
 /**
+ * The `$files` object extends the general
+ * `Collection` class and refers to a
+ * collection of files, i.e. images, documents
+ * etc. Files can be filtered, searched,
+ * converted, modified or evaluated with the
+ * following methods:
```

Comment:
I think we shouldn't do this. It's too much targeted at the online docs and no longer works when you read it in the code. Those blocks need to work in both ways or we need to create specific text blocks for our website if it's not possible in some cases. But we shouldn't be messing up our doc blocks.

Reply:
In what other context than the one described would this class ever be used? Imo it still works when you read the code. And it saves us in many places to maintain very similiar descriptions in 2 places.